Functions/Administration/xASL_adm_DefineASLSequence.m

```diff
@@ -5,87 +5,86 @@
 % FORMAT: [xQ] = xASL_adm_DefineASLSequence(xQ, bVerbose)
```

Comment:
Now we improve the nomenclature, shouldn't we call this function:

xASL_adm_DefineASLReadout

instead?

Reply:
It is still defining MRAcquisitionType and PulseSequenceType, so Sequence is equally valid as before. Readout isn't incorrect either, but I don't see why a change would be necessary.

Reply:
Because ASL sequence = labeling + readout. All this concerns the readout only. It's not a big deal, but if we 're changing the nomenclature anyway to more correct/clear, then readout is more clear than "sequence".

What sequence is it? A PCASL. Is said as more often than "What type of ASL sequence is it?" "2D_EPI"
So not (in)correct, just more clear.

I don't feel strong about this though :)

Reply:
I will update to readout.